multi/imxrt-multi/uart.c

```diff
@@ -107,13 +132,66 @@ static inline int uart_getTXcount(uart_t *uart)
 }
 
 
+static int gpio_setup(const uart_halfDuplexAction_t *action)
```

Comment:
Name prefix does not match the module name - we use prefixes to avoid name collisions. Static will always win, but it's undesirable nevertheless.

Reply:
Fixed